cmd/vicadmin/server.go

```diff
@@ -112,39 +127,208 @@ func (s *server) listenPort() int {
 	return s.l.Addr().(*net.TCPAddr).Port
 }
 
+// Enforces authentication on route `link` and runs `handler` on successful auth
+func (s *server) AuthenticatedHandle(link string, h http.Handler) {
```

Comment:
This and `Authenticated` seem to have exactly the same signature - presumably there's a reason why we have both?